openfermioncirq/gates/common_gates.py

```diff
@@ -253,3 +253,49 @@ def rot11(rads: float):
 FSWAP = FSwapPowGate()
 XXYY = XXYYPowGate()
 YXXY = YXXYPowGate()
+
+class CombinedSwapAndZ(
+        cirq.EigenGate,
+        cirq.InterchangeableQubitsGate,
+        cirq.TwoQubitGate):
+    """w0 * (XX + YY) + w1 * |11><11| interaction.
```

Comment:
I suggest using the convention `w0 * 0.5 * (XX + YY)` to match up with fermionic simulations, as in https://arxiv.org/pdf/1711.04789.pdf

Reply:
@kevinsung I changed the names, added support for complex weights, and reparameterized what is now called `QuadraticFermionicSimulationGate`. If you like that parameterization, I'll change `CubicFermionicSimulationGate` to match.